server/files/stylesheets/docs.css

```diff
@@ -1541,6 +1541,23 @@ body.progress.animated .ui.progress .bar {
   position: relative;
 }
 
+/*--------------
+      Emoji
+---------------*/
+#example emojiSearch.ui.search {
```

Comment:
Missing dot?
```suggestion
#example .emojiSearch.ui.search {
```

Reply:
No, it is referring to the implemented tag. It's not a class (i copied the logic from the icon module)
Related HTML code is in emoji.html.eco
```
    <emojiSearch class="ui right aligned search">
      <div class="ui icon input">
        <input class="prompt" type="text" placeholder="Search emojis...">
        <i class="search icon"></i>
      </div>
      <div class="results"></div>
    </emojiSearch>
```

Reply:
Ah, I see.